nixos/modules/programs/iio-hyprland.nix

```
@@ -0,0 +1,23 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  options = {
+    programs.iio-hyprland = {
+      enable = mkOption {
+        description = lib.mdDoc ''
+          Enable this option to get iio.hyprland and get/start iio-sensor-proxy.
+        '';
+        type = types.bool;
+        default = false;
+      };
```

```suggestion
      enable = mkEnableOption
        (mdDoc "iio-hyprland and iio-sensor-proxy");
```

---

`mdDoc` does nothing, I'd simply suggest

```suggestion
      enable = mkEnableOption "iio-hyprland and iio-sensor-proxy";      
```

---

I've done this without the mdDoc for now, if markdown is to be added I will add it to my module accordingly.